contracts/connectors/loantoken/LoanTokenLogicFactory.sol

```diff
@@ -0,0 +1,58 @@
+pragma solidity 0.5.17;
```

Comment:
why is this a factory? and why its extending LoanTokenLogicStandard

Reply:
This is the logic for factory ITokens and is the target for factory iTokens.